opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp

```diff
@@ -152,19 +141,14 @@ namespace Opm {
         for (size_t keywordIdx = 0; keywordIdx < section.size(); ++keywordIdx) {
             const auto& keyword = section.getKeyword(keywordIdx);
 
-            if (keyword.name() == "DATES") {
-                handleDATES(keyword);
```

Comment:
`handleDATES` and `handleTSTEP` are no longer used as far as I can see, so feel free to completely remove them. Good change!